### PR TITLE
fix(Google Drive Node): Fix regex in file RLC

### PR DESCRIPTION
**Pull request by michael-radency**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

**Review by Joffcom — DISMISSED** [review]
👍

**Review by netroy — COMMENTED** [review]
don't we need to update the v1 node as well ?

File: packages/nodes-base/nodes/Google/Drive/v1/GoogleDriveV1.node.ts [hunk 1]
```diff
@@ -243,14 +243,14 @@ const versionDescription: INodeTypeDescription = {
...
-							'https:\\/\\/(?:drive|docs)\\.google\\.com\\/\\w+\\/d\\/([0-9a-zA-Z\\-_]+)(?:\\/.*|)',
+							'https:\\/\\/(?:drive|docs)\\.google\\.com(?:\\/.*|)\\/d\\/([0-9a-zA-Z\\-_]+)(?:\\/.*|)',
 					},
 					validation: [
 						{
 							type: 'regex',
 							properties: {
 								regex:
-									'https:\\/\\/(?:drive|docs)\\.google.com\\/\\w+\\/d\\/([0-9a-zA-Z\\-_]+)(?:\\/.*|)',
+									'https:\\/\\/(?:drive|docs)\\.google.com(?:\\/.*|)\\/d\\/([0-9a-zA-Z\\-_]+)(?:\\/.*|)',
```
Comment (netroy) [inline packages/nodes-base/nodes/Google/Drive/v1/GoogleDriveV1.node.ts:253]: Shouldn't this be?

```suggestion
									'https:\\/\\/(?:drive|docs)\\.google\\.com(?:\\/.*|)\\/d\\/([0-9a-zA-Z\\-_]+)(?:\\/.*|)',
```

**Review by netroy — CHANGES_REQUESTED** [review]
While we are doing this, we should copy all of the 12 variants on this regexp string into a new file `packages/nodes-base/nodes/Google/constants.ts` as a single constant, instead of repeating this so many times.
This way we won't end up with multiple variants of this regexp string again.

**Comment by michael-radency** [comment]
> While we are doing this, we should copy all of the 12 variants on this regexp string into a new file `packages/nodes-base/nodes/Google/constants.ts` as a single constant, instead of repeating this so many times. This way we won't end up with multiple variants of this regexp string again.

thanks for suggestion @netroy, pushed update

**Review by netroy — APPROVED** [review]
LGTM

**Comment by Joffcom** [comment]
That was a good spot 🙌

**Review by Joffcom — APPROVED** [review]
Approving again just to be complete 🙏

**Pull request merged**